code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -123,3 +123,11 @@
 /mob/living/silicon/flash_act(intensity = 1, override_blindness_check = 0, affect_silicon = 0, visual = 0, type = /obj/screen/fullscreen/flash/static)
 	if(affect_silicon)
 		return ..()
+
+/mob/living/silicon/adjustToxLoss(amount, update_health, forced)
+	adjustBruteLoss(amount, update_health, forced)
+	return
+
+/mob/living/silicon/adjustOxyLoss(amount, update_health, forced)
+	adjustFireLoss(amount, update_health, forced)
```

Comment:
honestly oxygen damage should just regenerate on borgs, toxloss should go to fire